.vscode/launch.json

```diff
@@ -0,0 +1,33 @@
+{
+  "configurations": [
+    {
+      "name": "Jest $file",
+      "type": "node",
+      "request": "launch",
+      "runtimeArgs": [
+        "--inspect-brk",
+        "${workspaceRoot}/node_modules/jest/bin/jest.js",
+        "--testTimeout",
+        "100000",
+        "--runInBand",
+        "${file}"
+      ],
+      "cwd": "${workspaceFolder}",
+      "console": "integratedTerminal"
+    },
+    {
+      "name": "Run $file",
+      "type": "node",
+      "request": "launch",
+      "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/tsx",
```

Comment:
It's a cool shortcut. For this use case I usually just open a [JavaScript Debug Terminal](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_javascript-debug-terminal) and run a script manually. It magically auto-attaches to the process and even (some?) children processes